docs/cache.md

```diff
@@ -47,8 +47,43 @@ In this case you have to specify a custom factory in your `service_manager` conf
 ),
 ```
 
+Please be aware that you can't use *Closures* inside yout module configuration if you want to cache it. To avoid this problem, you have to use a Factory:
+```php
+// module.config.php
+'service_manager' => array(
+    'factories' => array(
+        'my_memcached_alias' => __NAMESPACE__ . '\Cache\MemcachedFactory'
```

Comment:
Config files usually do not have namespaces. Can you replace `__NAMESPACE__` with `YourModule`?